aion/web/elm/src/Panel/Notifications.elm

```diff
@@ -0,0 +1,21 @@
+module Panel.Notifications exposing (..)
+
+import General.Models exposing (Model)
+import General.Notifications exposing (addToast)
+import Msgs exposing (Msg(..))
+import Toasty.Defaults
+
+
+categoryFormValidationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
```

Comment:
Here I have a similar comment as to the previous PR. If we've put three functions mirroring three types of notifications in src/Notifications.elm, then this file could only consist of three strings + import Notifications. I think that would make it simpler, but, as I said, we can delay it until we'll be dealing with global elm refactor.